L1Trigger/L1TGlobal/interface/L1TGlobalUtil.h

```diff
@@ -202,11 +204,11 @@ namespace l1t {
 
     //file  and container for prescale factors
     std::string m_preScaleFileName;
-    unsigned int m_PreScaleColumn;
+    double m_PreScaleColumn;
```

Comment:
if this is the index of the prescale column, rather than the prescale value, it should stay as an `unsigned int` ?

Reply:
correct. reverting the developer's change.